test/unit/spec/function_output/modeling/msPathway-spec.js

```diff
@@ -1,23 +1,11 @@
-define(['msPathway', 'jquery'], (Widget, $) => {
+define(['msPathway', 'jquery', 'testUtil'], (Widget, $, TestUtil) => {
     'use strict';
+
     describe('Test the msPathway widget', () => {
+        afterAll(() => TestUtil.clearRuntime());
+
         it('Should load the module', () => {
             expect(ModelSeedPathway).toEqual(jasmine.any(Function));
         });
-
-        xit('Should do something or other', () => {
```

Comment:
Doesn't seem to do anything, and has been x'd out forever, so gone.

Reply:
I tried to fix it at some point but it seemed irreparably broken. Best deleted!